### PR TITLE
https://issues.redhat.com/browse/ACM-7839 Move OCP import

### PR DESCRIPTION
**Pull request by oafischer**

### CODE REVIEW

File: clusters/cluster_lifecycle/import_ocp.adoc [hunk 1]
```diff
@@ -1,7 +1,21 @@
...
+After you install {mce}, you are ready to import a cluster to manage. You can import an existing {ocp-short} cluster so that you can add additional nodes. Continue reading the following topics to learn more:
+
+* <<import-ocp-cluster-prereqs,Prerequisites>>
+* <<import-ocp-cluster-steps,Importing a cluster>>
+* <<add-nodes-ocp-infra-env,Adding worker nodes to {ocp-short} clusters>>
+
+[#import-ocp-cluster-prereqs]
+== Prerequisites
+
+* Draft
```
Comment (oafischer) [inline clusters/cluster_lifecycle/import_ocp.adoc]: @avishayt What are the prereqs for this procedure? This topic is now in the import section as requested, so the prereqs should look similar to the other import topics

File: clusters/cluster_lifecycle/import_ocp.adoc
Comment (oafischer) [inline clusters/cluster_lifecycle/import_ocp.adoc:1]: @elgnay @xiangjingli I would appreciate if you could look at this file per @berenss request. Thanks!

Comment (xiangjingli) [reply clusters/cluster_lifecycle/import_ocp.adoc:1]: This is owned by foundation squad. Le will review it

**Comment by elgnay** [comment]
LGTM

File: clusters/cluster_lifecycle/import.adoc [hunk 2]
```diff
@@ -7,10 +7,11 @@ A hub cluster cannot manage _any_ other hub cluster, but can manage itself. The
...
 
-Choose from the following instructions to set up your managed cluster, either from the console or from the CLI:
+Choose from the following instructions to set up your managed cluster:
 
 *Required user type or access level*: Cluster administrator
 
 * xref:../cluster_lifecycle/import_gui.adoc#importing-managed-cluster-console[Importing an existing cluster by using the console]
 * xref:../cluster_lifecycle/import_cli.adoc#importing-managed-cluster-cli[Importing a managed cluster by using the CLI]
 * xref:../cluster_lifecycle/import_agent.adoc#importing-managed-agent[Importing a managed cluster by using agent registration]
+* xref:../cluster_lifecycle/import_ocp.adoc#importing-managed-agent[Importing an {ocp} cluster]
```
Comment (avishayt) [inline clusters/cluster_lifecycle/import.adoc]: Maybe "Importing an on-premises OCP cluster" or "Importing a host inventory OCP cluster"?

File: clusters/cluster_lifecycle/import_ocp.adoc (hunk 1 above)
Comment (avishayt) [reply clusters/cluster_lifecycle/import_ocp.adoc]: The only prereq is enabling CIM

File: clusters/cluster_lifecycle/import_gui.adoc [hunk 3]
```diff
@@ -16,9 +16,7 @@ After you install {mce}, you are ready to import a cluster to manage. Continue r
 * A deployed hub cluster. If you are importing bare metal clusters, the hub cluster must be installed on {ocp} version 4.12 or later.
 * A cluster you want to manage.
 * The `base64` command line tool.
-* A defined `multiclusterhub.spec.imagePullSecret` if you are importing a cluster that was not created by {ocp-short}. This secret might have been created when {mce} was installed. See xref:../install_upgrade/adv_config_install.adoc#custom-image-pull-secret[Custom image pull secret] for more information about how to define this secret.
-+
-If you need to create a new secret, see <<creating-new-pull-secret,Creating a new pull secret>>.
+* A defined `multiclusterhub.spec.imagePullSecret` if you are importing a cluster that was not created by {ocp-short}. This secret might have been created when {mce} was installed. See _Custom image pull secret_ for more information about how to define this secret.
```
Comment (dockerymick) [inline clusters/cluster_lifecycle/import_gui.adoc:19]: The link for "Additional steps for running import commands manually in an {ocp-short} Dedicated environment"

File: clusters/cluster_lifecycle/import_ocp.adoc [hunk 4]
```diff
@@ -77,7 +91,7 @@ spec:
 ----
 +
 <1> Choose a name for your cluster.
-<2> Use `1` if you are using a single-node OpenShift (SNO) cluster. Use `3` if you are using a multinode cluster.
+<2> Use `1` if you are using a {sno} cluster. Use `3` if you are using a multinode cluster.
```
Comment (dockerymick) [inline clusters/cluster_lifecycle/import_ocp.adoc:94]: great catch!

**Review by dockerymick — APPROVED** [review]
Found a broken link, otherwise this looks good to me

File: clusters/cluster_lifecycle/import.adoc (hunk 2 above)
Comment (oafischer) [reply clusters/cluster_lifecycle/import.adoc]: Updated

File: clusters/cluster_lifecycle/import_ocp.adoc (hunk 1 above)
Comment (oafischer) [reply clusters/cluster_lifecycle/import_ocp.adoc]: Added

**Comment by oafischer** [comment]
Let me know if it looks good now @avishayt

**Comment by avishayt** [comment]
LGTM

**Pull request merged**